src/app/features/test-records/components/vehicle-header/vehicle-header.component.ts

```diff
@@ -47,7 +47,7 @@ export class VehicleHeaderComponent {
       case VehicleTypes.PSV:
         return techRecord.bodyMake && techRecord.bodyModel ? `${techRecord.bodyMake}-${techRecord.bodyModel}` : '';
       case VehicleTypes.HGV:
-        return techRecord.chassisMake && techRecord.chassisModel ? `${techRecord.chassisMake}-${techRecord.chassisModel}` : '';
+        return techRecord.make && techRecord.model ? `${techRecord.make}-${techRecord.model}` : '';
```

Comment:
I think Trailers use the same as a HGV if that's something that needs to be included on this ticket ?